<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqView</class>
 <widget class="QMainWindow" name="EqView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DREAM Equilibrium Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Details</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Magnetic axis (R&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;, Z&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMagneticAxis">
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Minor radius (a):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblMinorRadius">
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;On-axis magnetic field (B&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblB0">
           <property name="text">
            <string>n/a</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionShapingProfiles"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuQuantities">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionPlotJ"/>
    <addaction name="actionPlotPsi"/>
    <addaction name="actionPlotShaping"/>
    <addaction name="actionPlotG"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuQuantities"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShapingProfiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shaping profiles</string>
   </property>
  </action>
  <action name="actionPlotG">
   <property name="text">
    <string>Toroidal field function G(r)</string>
   </property>
  </action>
  <action name="actionPlotPsi">
   <property name="text">
    <string>Poloidal flux Ïˆ(r)</string>
   </property>
  </action>
  <action name="actionPlotShaping">
   <property name="text">
    <string>Shaping parameters</string>
   </property>
  </action>
  <action name="actionPlotJ">
   <property name="text">
    <string>Current density J(r)</string>
   </property>
  </action>
  <action name="actionWall">
   <property name="text">
    <string>Wall</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
